src/cli/cli.cpp

```diff
@@ -2733,6 +2745,18 @@ template <> otError Interpreter::Process<Cmd("log")>(Arg aArgs[])
 #if OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE
             uint8_t level;
 
+            /**
+             * @cli log level <level>
```

Comment:
This one is not getting imported via the reference script because OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE is not defined in our Doxygen config, you'll need to add it.

Make sure to run the reference import to validate that everything is coming through when sending for review.